master/reference/advanced/etcd-rbac/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+When using etcd it is a good idea to protect the data stored there.  This is
+even more true when you have multiple components using a common etcd cluster.
+
```

Comment:
Needs a sentence about what these guides are for (which is distinct from why you might be interested).